checkout-form-manager.js

```diff
@@ -394,11 +395,7 @@ function prefillForm(form) {
 
                 // Set pricing info
                 // TODO Get from data
-                _offerData = {
-                    "offer_id": "1358",
-                    "offer_name": "Essential Year Forecast 3 Month - $37",
-                    "offer_price": "37.00"
-                };
+                const _offerData = data['offerData'];
```

Comment:
@GauravBains  `_offerData` is an "instance variable" so it can be accessed by `updatePricing()`
```suggestion
                _offerData = data['offerData'];
```